infrastructure/constructs/my-lambda-view-app.ts

```diff
@@ -0,0 +1,50 @@
+import {Construct} from "constructs";
+
+import * as path from "path";
+import * as aws from "@cdktf/provider-aws";
+import MyLambda, {lambdaRolePolicy} from "./my-lambda";
+import MyApiLambdaIntegration from "./my-api-lambda-integration";
+import {MyApiGateway} from "./my-api-gateway";
+import MyDynamoDbTable from "./my-dynamo-db-table";
+
+export interface MyLambdaViewAppConfig {
+    name: string,
+    routeKey: string,
+    s3Bucket: aws.s3Bucket.S3Bucket,
+    viewData: MyDynamoDbTable
+    apiGateway: MyApiGateway,
+}
+
+
+export default class MyLambdaViewApp extends Construct {
```

Comment:
Instead of a separate ViewApp type, we could also just extend the existing App to allow access to view data? (However, both approaches are valid in my opinion. I think in the book, this is just treated as a "regular" app, i.e., apps can also access view data).